ftpd/index.d.ts

```diff
@@ -12,6 +12,14 @@ import fs = require("fs");
 import net = require("net");
 import tls = require("tls");
 
+export enum LogLevel {
```

Comment:
I would mark this as a `const` enum, as there is no exported _value_ named `LogLevel`.